Fedora-35/Dockerfile

```diff
@@ -87,6 +87,7 @@ RUN dnf \
         bzip2 \
         findutils \
         git \
+        lcov \
```

Comment:
This is redundant for the Fedora images since lcov is installed as part of the "build" layer the teat layer is based on.